typespec-tests/src/main/java/com/cadl/multipart/MultipartClient.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.cadl.multipart;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.ClientAuthenticationException;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.cadl.multipart.implementation.MultipartClientImpl;
+
+/** Initializes a new instance of the synchronous MultipartClient type. */
+@ServiceClient(builder = MultipartClientBuilder.class)
+public final class MultipartClient {
+    @Generated private final MultipartClientImpl serviceClient;
+
+    /**
+     * Initializes an instance of MultipartClient class.
+     *
+     * @param serviceClient the service client implementation.
+     */
+    @Generated
+    MultipartClient(MultipartClientImpl serviceClient) {
+        this.serviceClient = serviceClient;
+    }
+
+    /**
+     * request is binary.
+     *
+     * <p><strong>Request Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     data (Required): {
+     *         name: String (Required)
+     *         image: byte[] (Required)
+     *     }
+     * }
+     * }</pre>
+     *
+     * @param request The request parameter.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link Response}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> uploadWithResponse(BinaryData request, RequestOptions requestOptions) {
+        // Protocol API does not work, as operation 'upload' is 'multipart/form-data'
```

Comment:
Just say it won't work...

Reply:
If the protocol API does not work, why do we still generate it as `public`, probably make it package-private?

Reply:
make sense. change to internal as well.